tox.ini

```diff
@@ -7,7 +7,7 @@
 envlist = py27, py35, py37
 
 [testenv]
-commands = pytest --junitxml=test-results/junit-{envname}.xml
+commands = pytest -o junit_family=xunit2 --junitxml=test-results/junit-{envname}.xml
```

Comment:
Updating to new junit_family, related with: 
https://github.com/pytest-dev/pytest/issues/6179

Reply:
Can we make this a separate PR maybe? It definitely makes sense:

```console
.tox/py37/lib/python3.7/site-packages/_pytest/junitxml.py:436
  /home/daniel/dev/grafanalib/.tox/py37/lib/python3.7/site-packages/_pytest/junitxml.py:436: PytestDeprecationWarning: The 'junit_family' default value will change to 'xunit2' in pytest 6.0.
  Add 'junit_family=xunit1' to your pytest.ini file to keep the current format in future versions of pytest and silence this warning.
    _issue_warning_captured(deprecated.JUNIT_XML_DEFAULT_FAMILY, config.hook, 2)
```

Reply:
Moved it to https://github.com/weaveworks/grafanalib/pull/236